src/content/developers/tutorials/uniswap-v2-annotated-code/index.md

```diff
@@ -382,7 +382,7 @@ This function allows the factory (and only the factory) to specify the two ERC-2
 
 #### Internal Update Functions {#pair-update-internal}
 
-##### \_update {#_update}
+##### \_update {#\_update}
```

Comment:
Not sure why this change is here - what's the purpose?

Reply:
Hm, not sure. This appears to have been pulled in when merging `dev` back in, however, noticing that our markdown H5's don't parse these custom headers appropriately as is:

![image](https://user-images.githubusercontent.com/54227730/145067436-3b56a7c4-0a6a-432f-850d-627d4986b628.png)
![image](https://user-images.githubusercontent.com/54227730/145068002-b718bff1-93e7-4522-9300-55d7684ac93d.png)

I feel we should just remove these H5 custom header ids until we can address this issue separately. Can also remove the stray escape `\` that seems to be breaking the others, ie:
![image](https://user-images.githubusercontent.com/54227730/145069591-9e5697b4-7501-4afe-8d68-671fbc6a0b96.png)